tests/e2e/sqlalchemy/test_basic.py

```diff
@@ -181,7 +183,7 @@ def test_bulk_insert_with_core(db_engine, metadata_obj, session):
     )
 
     rows = [
-        {"name": names[i % 3], "number": random.choice(range(10000))}
+        {"name": names[i % 3], "number": random.choice(range(64))}
```

Comment:
This is necessary because DBSQL can only accept 255 parameters per execution, right now.